### PR TITLE
Master dev kit

### PR DESCRIPTION
**Pull request by ElectricMaxxx**
This one should fix the form type test.

### CODE REVIEW

**Comment by ElectricMaxxx** [comment]
To fix the last deprecation, https://github.com/doctrine/DoctrinePHPCRBundle/issues/285 should be done.

File: tests/Unit/Form/Type/SeoMetadataTypeTest.php [hunk 1]
```diff
@@ -14,24 +14,42 @@
...
  * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
  */
-class SeoMetadataTypeTest extends TypeTestCase
+class SeoMetadataTypeTest extends FormIntegrationTestCase
 {
+    private $validator;
+
+    public function setUp()
+    {
+        $this->validator = $this->createMock(ValidatorInterface::class);
```
Comment (ElectricMaxxx) [inline tests/Unit/Form/Type/SeoMetadataTypeTest.php]: we have to set it up on our own, as the base class in gone (moved to a trait) in symfony 3.4

File: src/Controller/SuggestionProviderController.php [hunk 2]
```diff
@@ -116,15 +119,6 @@ public function listAction(
     }
 
     /**
-     * @param SuggestionProviderInterface $matcher
-     * @param string                      $group
-     */
-    public function addSuggestionProvider(SuggestionProviderInterface $matcher, $group)
```
Comment (dbu) [inline src/Controller/SuggestionProviderController.php]: this is a BC break if somebody relied on adding providers later, e.g. in a compilar pass

File: src/DependencyInjection/CmfSeoExtension.php [hunk 3]
```diff
@@ -34,11 +34,6 @@ class CmfSeoExtension extends Extension
      */
     private $defaultAlternateLocaleProviderId;
 
-    /**
-     * @var bool Whether the content listener is loaded
-     */
-    private $contentListenerEnabled = false;
```
Comment (dbu) [inline src/DependencyInjection/CmfSeoExtension.php]: was this just unused?

Comment (ElectricMaxxx) [reply src/DependencyInjection/CmfSeoExtension.php]: yes, and sensio quality guard mentioned it.

File: src/Controller/SuggestionProviderController.php (hunk 2 above)
Comment (ElectricMaxxx) [reply src/Controller/SuggestionProviderController.php]: mhh, got a hint from sensio guard not to use other public methods then actions in controllers, what to do? Changelog enough?

Comment (ElectricMaxxx) [reply src/Controller/SuggestionProviderController.php]: ahh you wrote it :-)

Comment (ElectricMaxxx) [reply src/Controller/SuggestionProviderController.php]: Added a BC-BREAK notice in change log, i think we can live ẃith that on an non major release.

**Pull request merged**